features/issue/components/EditIssueForm.tsx

```diff
@@ -45,29 +55,63 @@ export const EditIssueForm = () => {
 		formState: { errors },
 	} = form
 
```

Comment:
const issue = await get(intissueId)

Potem w edit issue page robisz cos takieg

	<Suspense fallback={<Loading/>}>

	<EditIssueForm />
</Suspense>

tutaj przykład do tego https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#example

Reply:
Issue jest typu Issue | undefined.

Jeżeli jest undefined wyświtlasz błąd albo jakiś button do powrotu

Jeżeli jest zdefiniowany to wyświetlasz forma i wszzystko git